tests/api.v2/middlewares/authMiddlewares.test.js

```diff
@@ -92,15 +99,41 @@ describe('Tests for authorization/authentication middlewares', () => {
     expect(next).toBeCalledWith(expect.any(UnauthenticatedError));
   });
 
+  it('Express middleware can reject users that didnt agree to privacy policy', async () => {
```

Comment:
`didn't`

Reply:
I don't add the ' in the tests titles because the linter auto changes them to \' . When that happens and e.g. I get informed that "policy wasn't agreed on" failed I can't just copy paste and search for it directly because I need to first add the escape characters to the title. If I don't add the ' all this problem disappears